aspnetcore/fundamentals/minimal-apis/includes/webapplication.md

```diff
@@ -0,0 +1,254 @@
+## `WebApplication`
+
+The following code is generated by an ASP.NET Core template:
+
+[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_default)]
+
+The preceding code can be created via `dotnet new web` on the command line or selecting the Empty Web template in Visual Studio.
+
+The following code creates a <xref:Microsoft.AspNetCore.Builder.WebApplication> (`app`) without explicitly creating a <xref:Microsoft.AspNetCore.Builder.WebApplicationBuilder>:
+
+[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_create)]
+
+[`WebApplication.Create`](xref:Microsoft.AspNetCore.Builder.WebApplication.Create%2A) initializes a new instance of the <xref:Microsoft.AspNetCore.Builder.WebApplication> class with preconfigured defaults.
+
+### Working with ports
+
+When a web app is created with Visual Studio or `dotnet new`, a `Properties/launchSettings.json` file is created that specifies the ports the app responds to. In the port setting samples that follow, running the app from Visual Studio returns an error dialog `Unable to connect to web server 'AppName'`. Run the following port changing samples from the command line.
```

Comment:
```suggestion
When a web app is created with Visual Studio or `dotnet new`, a `Properties/launchSettings.json` file is created that specifies the ports the app responds to. In the port setting samples that follow, running the app from Visual Studio returns an error dialog `Unable to connect to web server 'AppName'`.  Visual Studio returns an error because it's expecting the port specified in `Properties/launchSettings.json`, but the app is using the port specified by `app.Run("http://localhost:3000")`. Run the following port changing samples from the command line.
```